jerry-core/lit/lit-strings.c

```diff
@@ -243,6 +243,29 @@ convert_code_point_to_high_surrogate (lit_code_point_t code_point) /**< code poi
   return (LIT_UTF16_HIGH_SURROGATE_MARKER | code_unit_bits);
 } /* convert_code_point_to_high_surrogate */
 
+/**
+ * UTF16 Encoding method for a code point
+ *
+ * See also:
+ *          ECMA-262 v6, 10.1.1
+ *
+ * @return uint8_t, the number of returning code points
+ */
+uint8_t
+ecma_builtin_string_utf16_encode_code_point (lit_code_point_t cp, /**< the code point we encode */
```

Comment:
Since this is now in `lit-strings.c`, could we rename it to `lit_utf16_encode_code_point` or something similar?